### ISSUE TITLE
AbsRel value

### ISSUE THREAD

**Issue by AmandineBtto** [AmandineBtto/Batvision-Dataset#2]
兄弟，你有遇到过absrel很高的情况嘛，其他指标都很正常

_Originally posted by @yinjun-0618 in https://github.com/AmandineBtto/Batvision-Dataset/issues/1#issuecomment-2008352335_

**Comment by AmandineBtto** [comment]
Hello @yinjun-0618,

I have opened a new issue dedicated to your problem so I can help you further. 
I'm sorry but I don't speak Chinese. Would it be possible for you to translate to English?

As far as I could understand with online translator, you have encountered high abs rel value is that right? Are you using the same config? I don't remember having this problem but with more details maybe I can help you.

**Comment by Kang-ChangWoo** [comment]
Hello @AmandineBtto,

Thanks for sharing the BatVision dataset and code.

I’m facing the same issue: in BatVision v2 audio-only, all other metrics match the paper, but ABS_REL is about 2× higher.  
Configs and data splits are identical to the released code.

Same config and data split as the released code. 

Could you help check this?

**Comment by AmandineBtto** [comment]
Hello @Kang-ChangWoo 

I'm not sure I'll be able to help as I'm not working on this project since a long time, but I'll try my best. 
Did you use the "test.py" file for evaluation ? and the metric computation from the repo?

**Comment by Kang-ChangWoo** [comment]
@AmandineBtto ,

Thank you for the quick reply.
I understand what you mean. 😄 

I execute `train.py` and print the metrics during validation to monitor performance. 
I also execute `test.py` to evaluate the pretrained models (their results are similar).

Below image shows that results from the running `test.py` with the pretrained checkpoint.

<img width="103" height="97" alt="Image" src="https://github.com/user-attachments/assets/b1e9a162-a8a7-43b6-8ed6-d177728fcc1c" />

I use the metric computation from your repo!

**Comment by Kang-ChangWoo** [comment]
That the default setting of `cv2.resize()` function is bilinear interpolation in [utils_dataset.py](https://github.com/AmandineBtto/Batvision-Dataset/tree/main/UNetSoundOnly/dataloader) seems to have caused issues. Switching to nearest gave more consistent results. Zero-value regions appear to have corrupted the depth.

Below is an example image.
<img width="797" height="233" alt="Image" src="https://github.com/user-attachments/assets/e1ce3d94-644f-4235-8843-ecbdc971fdf4" />

By doing so, I can get below metrics.

<img width="664" height="15" alt="Image" src="https://github.com/user-attachments/assets/051091fd-1929-4832-9433-f3f8e6946450" />
